addon/globalPlugins/dengjen_tts_global_plugin/voice_manager.py

```
@@ -296,6 +297,9 @@ def _get_process_tar_archive(self, dialog, res):
 
 
 class OnlineDengjenVoicesPanel(SizedPanel):
+    # Translators: label of a button
+    _PREVIEW_LABEL = _("&Preview")
```

Ruff (0.16.2)

[error] Undefined name `_`

(F821)